tests/testthat/test-estimateParams.R

```diff
@@ -1,6 +1,7 @@
 # prepareData ----
 
 test_that("prepareData works as expected", {
+  local_edition(3)
```

Comment:
I think better to use the 3rd edition consistently, you can set this up with running `usethis::use_testthat(edition = 3)`